features/hbx_admin/ui_config/staff.feature

```diff
@@ -0,0 +1,31 @@
+Feature: Staff tab
```

Comment:
minor comments
instead of having multiple Given's use `Given` after that use `And`
Also use `When` before `Then`, I saw some scenarios we have `Then` without `When`.